static/js/teachers.ts

```diff
@@ -16,7 +16,7 @@ export function create_class() {
       contentType: 'application/json',
       dataType: 'json'
     }).done(function(_response) {
-      location.reload ();
+      window.location.pathname = '/customize-class/' + _response.id ;
```

Comment:
Not specifically a comment on this PR, but this makes me notice the URL does not start with `for-teachers`.

If it did, we would automatically properly detect and render the main menu appropriately (with the "for teachers" tab selected).

So maybe worth changing?

Reply:
Not sure how we would change this, do you mean `for-teachers/customize-class` ?

Reply:
That's what I meant. You can also say "neh", it was just an idea.

You will start to see the difference after combining with this commit: https://github.com/Felienne/hedy/commit/42ab2641d2c26a101a870371e7c16f0de8729439

Reply:
I agree that this would be better! Other than the menu bar issue we might also implement more pages such as `customize-student` that would belong to the `for-teachers` section. Makes more sense to group them there. Is it okay if I pick this up in a different issue?

Reply:
Yeah it has the benefit of the menu bar being correct "for free" But sure, let's do that separately!